<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TipsDialog</class>
 <widget class="QDialog" name="TipsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Did you know?</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QCheckBox" name="startupCheckBox">
     <property name="text">
      <string>Show tips at startup</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTextBrowser" name="tipsBrowser">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="go-next"/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="previousButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="go-previous"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
